docs/DeveloperGuide.md

```diff
@@ -1066,8 +1066,30 @@ testers are expected to do more *exploratory* testing.
 
 #### View student information
 
-1. _{ more test cases to come …​ }_
-
+1. View all related information of a student
+  1. Prerequisite: List all student using the `list` command. Multiple students are shown in the list. **List should have exactly 6 students** (can using sample data provided as default)
+  2. Test case: `view 1`
+     Expected: All related information of the first student is shown on the result display visualizer on the right side. This includes: Student name, current tags, all academic-related information (assessment score, studio participation, studio attendance), and personal contact information (phone, email, telegram)
+     Expected: No information is modified, Academy Directory runs as normal
+     Expected: Status message is that users are viewing student at position 1 of the list
+  3. Test case: `view 6`
+     Expected: All related information of the last student is shown on the result display visualizer on the right side. This includes: Student name, current tags, all academic-related information (assessment score, studio participation, studio attendance), and personal contact information (phone, email, telegram)
+     Expected: No information is modified, Academy Directory runs as normal
+     Expected: Status message is that users are viewing student at position 6 of the list
+  4. Test case: `view 7`
+     Expected: No view is shown on the result display. An error message is shown stating that index number is invalid
+  5. Test case: `view 0`, `view add`, `view myself in front of the mirror as a failure of society`
+     Expected: No view is shown on the result display. An error message is shown stating that index number is invalid (in a sense that it must be a positive integer)
+
+2. View students when list is altered
+  1. Prerequisite: Using filter to reduce the list view to 1 only. List should only have one student filtered.
```

Comment:
Might want to specify the filter command used to filter since you're working with the sample data provided as default. e.g `filter alex` would produce a `list` of length 1 to remove confusion.